src/tests/end-to-end/solver-call.cpp

```diff
@@ -161,4 +168,34 @@ BOOST_AUTO_TEST_CASE(free_data_sample)
 	filesystem::remove_all(studyPath + "logs/");
 }
 
+//Test free data sample study objective functions result
+BOOST_AUTO_TEST_CASE(free_data_sample_ortools_sirius)
+{
+	//First check if sirius is available with ortools
+	if (!OrtoolsUtils().isOrtoolsSolverAvailable(Antares::Data::OrtoolsSolver::sirius))
+	{
+		return;
+	}
+
+	const std::string& studyPath		= getBuildDirectory() + "/tests/data/free_data_sample/";
+
+	const std::vector<double>& checkIntegrityExpectedValues = { 2.85657392370263e+11,
+																0.00000000000000e+00,
+																2.85657392370263e+11,
+																2.85657392370263e+11,
+																2.85657544872729e+11,
+																0.00000000000000e+00,
+																2.85657544872729e+11,
+																2.85657544872729e+11 };
```

Comment:
indentation

Reply:
we need to define if we use space or tab in our indentation